website/package.json

```diff
@@ -9,7 +9,7 @@
     "rename-version": "docusaurus-rename-version"
   },
   "devDependencies": {
-    "docusaurus": "1.0.15"
+    "docusaurus": "1.1.2"
```

Comment:
We ran into a problem of npm outage causing 1.1.2 to not be downloadable. We just released 1.1.3 as a workaround even though there aren't much changes. Could you use 1.1.3 just to be safe? Sorry and thanks!